hutch_python/plan_defaults.py

```diff
@@ -18,22 +17,23 @@ def collect_plans(modules):
         for name, obj in module.__dict__.items():
             try:
                 # Only include things that are natively from this module
-                if obj.__module__ == module_name:
-                    try:
-                        # Check the __wrapped__ attribute for decorators
-                        if isgeneratorfunction(obj.__wrapped__):
-                            plans[name] = obj
-                    except AttributeError:
-                        # Not a decorator, check obj
-                        if isgeneratorfunction(obj):
-                            plans[name] = obj
+                from_module = obj.__module__ == module_name
+                # Only include callables
+                is_callable = callable(obj)
+                # Skip hidden items
+                is_hidden = len(name) == 0 or name[0] == '_'
+
+                if from_module and is_callable and not is_hidden:
+                    plans[name] = obj
             except AttributeError:
                 # obj did not have __module__, probably a builtin
                 pass
     return SimpleNamespace(**plans)
 
 
-plans = collect_plans(['bluesky.plans'])
-plan_stubs = collect_plans(['bluesky.plan_stubs'])
+plans = collect_plans(['bluesky.plans',
+                       'nabs.plans'])
```

Comment:
Worth noting: modules listed later in the list will override plan entries from the previous ones.

The ordering shown makes sense, as we'd want our custom `nabs` plans to override anything built-in from bluesky.